### PR TITLE
[DOC]Redirect to magma.github.io

### PR DESCRIPTION
**Pull request by tmdzk**
## Summary

Redirect documentation to the new github pages

## Test Plan

Go to https://facebookincubator.github.io/magma/docs/lte/setup_deb and make sure it redirects you to https://magma.github.io/magma/docs/basics/introduction.html

# Additional Information

Need to disable the docusaurus job on facebookincubator

### CODE REVIEW

**Review by xjtian — APPROVED** [review]
oh neat I didn't think it'd be this simple

**Comment by xjtian** [comment]
let's disable the docusaurus bot from OSS dashboard first before merging this @tmdzk

**Comment by tmdzk** [comment]
@xjtian https://github.com/facebookincubator/magma/pull/2008. Docusaurus-bot is just a user with write access, CI is using its token to push to gh-pages (where github.pages fetch the website's code) just removing the docusaurus worflow would stop the autodeploy

**Pull request merged**
